aiida/orm/nodes/process/calculation/calcjob.py

```diff
@@ -41,6 +41,7 @@ class CalcJobNode(CalculationNode):
     # pylint: disable=too-many-public-methods
 
     CALC_JOB_STATE_KEY = 'state'
+    CALC_JOB_STATE_TRACKER = 'state_changes'
```

Comment:
I would append `_KEY` to the name. Also, once we define the name for the config option, we might want to pick a different name here as well (maybe)